packages/lib/src/core/core.ts

```diff
@@ -330,7 +342,9 @@ class Core {
      */
     private handleCreateError(paymentMethod?): never {
         const paymentMethodName = paymentMethod && paymentMethod.name ? paymentMethod.name : 'The passed payment method';
-        const errorMessage = paymentMethod ? `${paymentMethodName} is not a valid Checkout Component` : 'No Payment Method component was passed';
+        const errorMessage = paymentMethod
+            ? `${paymentMethodName} is not a valid Checkout Component. (What was passed:${JSON.stringify(paymentMethod)})`
+            : 'No Payment Method component was passed';
```

Comment:
If I am not mistaken, most of the times this error happens is because merchants try to instantiate a payment method but it is not being returned as part of the /paymentMethods response (maybe it wasn't configured properly on BO).

So it could be that it is a valid Checkout component, but it is not configured in the merchant account.

Maybe we can enhance the message and say something like : 
`Failed to create 'txvariant' . Check if this payment is configured in the Backoffice or if the txVariant is a valid one.`